packages/datasource/contentful/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import { GetMeshSourceOptions, YamlConfig } from '@graphql-mesh/types';
+const GraphQLHandler = require('@graphql-mesh/graphql');
+
+export default class ContentfulHandler extends GraphQLHandler {
+  constructor({ name, config, cache }: GetMeshSourceOptions<YamlConfig.GraphQLHandler>) {
+    // @ts-ignore
+    config.schemaHeaders = { authorization: `Bearer ${config.token}`};
```

Comment:
Some formatting is missing (trailing spaces after "`" and "}" ). Probably Prettier is disabled.

Reply:
+